cap/modules/deposit/serializers/schemas/json.py

```diff
@@ -52,34 +51,28 @@ class DepositFormSchema(DepositSchema):
     """Schema for deposit v1 in JSON."""
 
     schemas = fields.Method('get_deposit_schemas', dump_only=True)
-
     webhooks = fields.Method('get_webhooks', dump_only=True)
     workflows = fields.Method('get_workflows', dump_only=True)
 
     can_update = fields.Method('can_user_update', dump_only=True)
     can_admin = fields.Method('can_user_admin', dump_only=True)
 
     def get_webhooks(self, obj):
-        deposit = CAPDeposit.get_record(obj['pid'].object_uuid)
-        webhooks = deposit.model.webhooks
-
+        webhooks = obj['deposit'].model.webhooks
         return GitWebhookSubscriberSchema(many=True).dump(webhooks).data
 
     def get_workflows(self, obj):
-        deposit = CAPDeposit.get_record(obj['pid'].object_uuid)
-        workflows = deposit.model.reana_workflows
-
+        workflows = obj['deposit'].model.workflows
```

Comment:
Sorry again, but this piece doesn't work :(

`RecordMetadata` model has a backref to workflows model by `reana_workflows` (you can see how we define it [here](https://github.com/cernanalysispreservation/analysispreservation.cern.ch/blob/404a1c7e705e9f651de8c914336a9e63d1c9c7ce/cap/modules/workflows/models.py#L76))

In general, if you're not sure what some pieces of code do, just ask or try not to change them, if not necessary